docs/readmes/orc8r/upgrade_1_4.md

```diff
@@ -50,16 +50,15 @@ container versions
 module orc8r {
   source = "github.com/magma/magma//orc8r/cloud/deploy/terraform/orc8r-aws?ref=v1.4"
   # ...
-
   cluster_version = "1.17"   # set to Kubernetes version found above. 1.17 is used as an example here
 }
 
 module orc8r-app {
   source = "github.com/magma/magma//orc8r/cloud/deploy/terraform/orc8r-helm-aws?ref=v1.4"
   # ...
   orc8r_chart_version   = "1.5.12"
-  orc8r_tag             = "MAGMA_TAG"       # from build step, e.g. v1.4.0
-  orc8r_deployment_type = DEPLOYMENT_TYPE   # valid options are: ["fwa", "federated_fwa", "all"]
+  orc8r_tag             = "MAGMA_TAG"  # from build step, e.g. v1.4.0
+  orc8r_deployment_type = "all"        # valid options: ["fwa", "federated_fwa", "all"]
```

Comment:
I think using "fwa" here might be better. I don't foresee an operator needing to deploy all.